src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToStartHandlerTest.java

```diff
@@ -246,7 +247,26 @@ public void givenAppealWithHearingRecordingFeatureFlagOn_populateDirectionTypeDr
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
 
         DynamicList expected = new DynamicList(new DynamicListItem("", ""), listOptions);
-        assertEquals(expected, response.getData().getDirectionTypeDl());
         assertEquals(3, listOptions.size());
+        assertEquals(expected, response.getData().getDirectionTypeDl());
+    }
+
+    @Test
+    public void givenAppealWithNoHearingRecordingRequestOutstanding_populateDirectionTypeDropdownWithRefuseHearingRecordingRequest() {
```

Comment:
the test name suggests it will populate a direction type with a Refuse Hearing Recording request, but the test does not do that. It looks like the name of the test is incorrect.